optimize/self-managed/optimize-deployment/configuration/shared-elasticsearch-cluster.md

```diff
@@ -1,21 +1,23 @@
 ---
 id: shared-elasticsearch-cluster
-title: "Shared Elasticsearch cluster"
-description: "Operate multiple Optimize instances on a shared Elasticsearch cluster."
+title: "Shared Elasticsearch/OpenSearch cluster"
```

Comment:
I am a little bit worried that this changes imply OpenSearch support for C7 customers as well, because they share the docs